modules/appeals_api/app/controllers/appeals_api/application_controller.rb

```diff
@@ -3,5 +3,10 @@
 module AppealsApi
   class ApplicationController < ::ApplicationController
     skip_before_action :set_tags_and_extra_context, raise: false
+
+    def render_response(resp)
+      response.headers = resp.headers
```

Comment:
we should probably remove the lighthouse api key from the headers we pass along

Reply:
whoops! yes we should (although I believe I'll do it in the contestable_issues_controller or service.rb)

Reply:
or, it sounds like this is stripped out before it hits vets-api